lib/text/ui_text_displayer.js

```diff
@@ -158,9 +172,10 @@ shaka.text.UITextDisplayer = class {
 
   /**
    * Display the current captions.
+   * @param {boolean=} force
    * @private
    */
-  updateCaptions_() {
+  updateCaptions_(force = true) {
```

Comment:
LGTM, thank you! Is there a more concrete name than "force", like "forceUpdate" or "forceRemoval" ?

Reply:
Done!